content/docs/studio/troubleshooting.md

```diff
@@ -1,3 +1,12 @@
+<admon>
+
+**We have renamed Views to Projects in Iterative Studio.**
+
+Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
+settings_ are now called _Project settings_; and so on.
+
+</admon>
+
```

Comment:
Probably not needed in this page either? Up to you

```suggestion
```

Reply:
Not removing it from this page. People might come to this page to figure out why they don't see their views. So, it would be good to notify them clearly.